datasetconfig.json

```diff
@@ -121,6 +124,9 @@
             "isnowvol": { "name": "Snow Volume", "envtype": "ice", "unit": "cm", "scale": [0, 2000] },
             "itmecrty": { "name": "Sea Ice Y Velocity", "envtype": "ice", "unit": "m/s", "scale": [-1, 1], "zero_centered": "true" },
             "itzocrtx": { "name": "Sea Ice X Velocity", "envtype": "ice", "unit": "m/s", "scale": [-1, 1], "zero_centered": "true" },
```

Comment:
I suppose for consistency we should remove itmecrty and itzocrtx for this dataset as well?

Reply:
Actually, now I'm getting worried about removing all of the vozocrtx/vomecrty because vozocrte/vomecrtn use them in their equations. Is there an option to "hide/disable" those variables instead? What impact would that have on using them in calculations but not displaying them in the list of variables?